### PR TITLE
DO NOT MERGE. FOR TESTING ONLY 2.0

### PR DESCRIPTION
**Pull request by eschutho**
SUMMARY
this is for testing only. The list of cherries is here for reference to see what can be tested:
https://github.com/apache/superset/issues?q=label%3Av2.0.1

TESTING INSTRUCTIONS
Some features for database connections will not work because not all drivers are installed on the testing environment. Do not add any databases with sensitive information, and remove the db when finished testing.

### CODE REVIEW

**Review by AAfghahi — CHANGES_REQUESTED** [review]
by request

File: superset-frontend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "superset",
-  "version": "2.0.0",
+  "version": "0.0.0-dev",
```
Comment (eschutho) [inline superset-frontend/package.json:3]: @AAfghahi it looks like this changed
```suggestion
  "version": "2.0.1",
```

File: superset-frontend/package.json [hunk 2]
```diff
@@ -113,7 +117,7 @@
     "@vx/responsive": "^0.0.195",
     "abortcontroller-polyfill": "^1.1.9",
     "ace-builds": "^1.4.14",
-    "antd": "^4.9.4",
+    "antd": "4.10.3",
```
Comment (eschutho) [inline superset-frontend/package.json:120]: we should probably unpin this while we're here

**Pull request closed without merging**